src/app/organizer/Columns.tsx

```diff
@@ -339,15 +339,18 @@ export function getColumns(
         header: t('Organizer.Columns.ModSlot'),
         // TODO: only show if there are mod slots
         value: (item) =>
-          getSpecialtySocketMetadatas(item)
+          getInterestingSocketMetadatas(item)
             ?.map((m) => m.slotTag)
             .join(','),
         cell: (value, item) =>
-          value && <SpecialtyModSlotIcon className={styles.modslotIcon} item={item} />,
-        filter: (_val, item) => {
-          const modSlotMetadata = getSpecialtySocketMetadatas(item);
-          return `modslot:${modSlotMetadata?.[0].slotTag || 'none'}`;
-        },
+          value && (
+            <SpecialtyModSlotIcon
+              className={styles.modslotIcon}
+              item={item}
+              excludeStandardD2ModSockets
+            />
+          ),
+        filter: (value) => (value !== undefined ? `modslot:${value}` : ``),
```

Comment:
`value`, as defined above, will be a comma separated list of tag strings, like `combat,vog`
so this will generate an invalid filter if there's more than one "interesting" slot.

now, there probably *isn't* more than one, but bungie loves to revamp the mod slots :)

this could be fixed up by
re-processing the item (`filter: (_val, item) =>`)
or text splitting the `value`

and then mapping it to multiple potential `modslot:` filters,

but i'd like to see this potential case covered so we aren't generating broken filters later